src/TwitchLib.Communication.Tests/TwitchLib.Communication.Tests.csproj

```diff
@@ -1,15 +1,22 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
I would stay on .net 6, who knows what will happen to other libraries if this suddenly changes to .NET 7. Also it breaks automatic checks here.

Reply:
Yeah. I saw that within the checks that failed and I'm going to revert it

Reply:
In theory it wouldnt be an issue bcs thats just the Test project nobody should really upon but I guess I would keep it at .NET 6 as that is the LTS right now